Makefile

```diff
@@ -0,0 +1,16 @@
+setup: requirements.txt requirements-dev.txt requirements-test.txt
+	python3 -m venv venv
+
+	. venv/bin/activate && \
+	pip3 install -r requirements.txt && \
+	pip3 install -r requirements-dev.txt && \
+	pip3 install -r requirements-test.txt && \
+	pip3 install -e . && \
+	pre-commit install && \
+	pre-commit run
```

Comment:
Since this is for dev, I believe we need to include the `requirements-reports.txt` and `requirements-ml.txt` for the install as well.